docs/developers/keystore.md

```diff
@@ -0,0 +1,56 @@
+---
+id: keystore
+title: MultiversX Smart Contracts
+---
+
+[comment]: # (mx-abstract)
+
+A keystore is a mechanism for storing private keys. It is a JSON file that encrypts a private key and is the standard for interchanging keys between devices as until a user provides their password, their key is safe.
```

Comment:
`secret keys or mnemonics (seed phrases)` - we don't use the term `private keys` too often (generally speaking).

"devices as until a user provides" - a sentence end seems to be missing?